docs/approach.rst

```diff
@@ -149,14 +156,25 @@ of the other echoes (which it is).
     We do, however, make it accessible as an alternative combination method
     in the t2smap workflow.
 
+Denoising
+`````````
+The next step is an attempt to remove noise from the data. This process can be 
+broadly seperated into three steps: **decomposition, metric calculation** and 
+**component selection**. Decomposition reduces the dimensionality of the 
+optimally combined data using PCA and then an ICA. Metrics which highlights the
```

Comment:
Should we define PCA and ICA ? Maybe add links to the sklearn docs ?

Reply:
Also, do we have a list of metrics anywhere ?

Reply:
for first - good idea. 

For the second - no, but would be nice. But I think this needs to wait on the "minimal decision tree A" etc etc bit.

Reply:
Cause that would mesh well with me in something that was sorted like the outputs section...once its done.